extensions/amp-sidebar/0.1/test-e2e/test-amp-sidebar.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {Key} from '../../../../build-system/tasks/e2e/functional-test-controller';
+import {Key} from '../../../../build-system/tasks/e2e/functional-test-types';
```

Comment:
A thought - if we get rid of puppeteer-controller, we could export `Key` as part of selenium-web-controller and avoid having to import this in individual tests.

Reply:
I love this idea! I'd rather do this as a separate PR though as this one is already getting pretty big.

Reply:
Yep totally, just mentioned as a follow up